test/parallel/test-timers-destroyed.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+
+const t1 = setTimeout(common.mustNotCall(), 1);
+const t2 = setTimeout(common.mustCall(), 1);
+const i1 = setInterval(common.mustNotCall(), 1);
+const i2 = setInterval(common.mustCall(), 1);
+i2.unref();
+
+// Keep process alive for i1 to call once due to timer ordering.
```

Comment:
```suggestion
// Keep process alive for i2 to call once due to timer ordering.
```

(I assume)

Reply:
I think `i1` is correct: the process ends before i2 is destroyed. So it's only kept alive as long as `i1` is done (`i2` is unrefed, that's why the process does not end on it before exiting).

Reply:
Nah, `i1` is cleared later. Anna is correct.